CHANGELOG.rst

```diff
@@ -2,6 +2,10 @@
 Changelog
 =========
 
+2.0.3 (2024-12-10)
+------------------
```

Comment:
```suggestion
3.0.0 (2024-12-XX)
------------------
```

We'll also be updating the code to work with Numpy 2.0.  Targeting the release date for Dec, but this will be tricky with the holidays.